docs/src/examples/components/Menu/Variations/index.tsx

```diff
@@ -49,6 +49,16 @@ const Variations = () => (
       description="A vertical menu may take the size of its container."
       examplePath="components/Menu/Variations/MenuExampleVerticalFluid"
     />
+    <ComponentExample
+      title="Menu as a Tab List"
+      description="A menu that acts as a tablist."
```

Comment:
let's change the description to "A menu with TabList accessibility behavior" and the one below to "A menu with toolbar accessibility behavior"